js-weather/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8">
+  <title>天気JS</title>
+</head>
+
+<body>
+    <h3>各国の天気</h3>
+    <select id="area">
+      <option value="London" selected>ロンドン</option>
+      <option value="Tokyo">東京</option>
+      <option value="Paris">パリ</option>
+      <option value="New York">ニューヨーク</option>
+      <option value="Indonesia">インドネシア</option>
+    </select>
+    <h3 id="desc"></h3>
```

Comment:
見た目的には仕様を満たせているのですが、正しいマークアップを意識しましょう。
h3は見出しを意味するので、動的に結果を書き換える要素のマークアップには不向きです。

意味の持たない、`span`や`div`が良いですね。

```suggestion
    <span id="desc"></h3>
```